src/components/BookingBreakdown/LineItemRefundMaybe.js

```diff
@@ -1,23 +1,62 @@
 import React from 'react';
 import { FormattedMessage, intlShape } from 'react-intl';
+import Decimal from 'decimal.js';
 import { formatMoney } from '../../util/currency';
-import { propTypes } from '../../util/types';
+import { types as sdkTypes } from '../../util/sdkLoader';
+import config from '../../config';
+import {
+  propTypes,
+  LINE_ITEM_CUSTOMER_COMMISSION,
+  LINE_ITEM_PROVIDER_COMMISSION,
+} from '../../util/types';
 
 import css from './BookingBreakdown.css';
 
-const LineItemRefundMaybe = props => {
-  const { transaction, unitType, intl } = props;
+const { Money } = sdkTypes;
+
+/**
+ * Calculates the total price in sub units for multiple line items.
+ */
+const lineItemsTotal = lineItems => {
+  const amount = lineItems.reduce((total, item) => {
+    return total.plus(item.lineTotal.amount);
+  }, new Decimal(0));
+  return new Money(amount, config.currency);
+};
 
-  const refund = transaction.attributes.lineItems.find(
-    item => item.code === unitType && item.reversal
+/**
+ * Checks if line item represents commission
+ */
+const isCommission = lineItem => {
+  return (
+    lineItem.code === LINE_ITEM_PROVIDER_COMMISSION ||
+    lineItem.code === LINE_ITEM_CUSTOMER_COMMISSION
   );
+};
+
+/**
+ * Returns non-commission, reversal line items
+ */
+const nonCommissionReversalLineItems = transaction => {
+  return transaction.attributes.lineItems.filter(item => !isCommission(item) && item.reversal);
+};
+
+const LineItemRefundMaybe = props => {
+  const { transaction, intl } = props;
+
+  // all non-commission, reversal line items
+  const lineItems = nonCommissionReversalLineItems(transaction);
```

Comment:
Maybe this could be renamed as `const refundLineItems = `?

Reply:
@Gnito This follows the LineItemSubTotalMaybe component in naming the line items const but maybe I could rename in both.